client/client.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"context"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+
+	// This import path is based on the name declaration in the go.mod,
+	// and the gen/proto/go output location in the buf.gen.yaml.
+	searchv1 "github.com/JacobPuff/github-search-api/gen/proto/go/githubsearch/v1"
+	"google.golang.org/grpc"
+)
+
+var SEARCH_TERM = GetEnvOrDefault("SEARCH_TERM", "search repo:jacobpuff/github-search-api")
+var USER = GetEnvOrDefault("USER", "")
```

Comment:
```suggestion
var User = GetEnvOrDefault("USER", "")
```
Why make these globals rather than local to the run function like the other env vars